simapp/app.go

```diff
@@ -188,7 +188,7 @@ func NewSimApp(
 	app.CrisisKeeper = crisis.NewKeeper(
 		app.subspaces[crisis.ModuleName], invCheckPeriod, app.SupplyKeeper, auth.FeeCollectorName,
 	)
-	app.UpgradeKeeper = upgrade.NewKeeper(skipUpgradeHeights, keys[upgrade.StoreKey], app.cdc)
+	app.UpgradeKeeper = upgrade.NewKeeper(skipUpgradeHeights, keys[upgrade.StoreKey], codec.NewHybridCodec(app.cdc))
```

Comment:
FYI I just used `codec.NewHybridCodec(app.cdc)` for now to keep things simple. When #5533 gets merged this can switch to `appCodec.

Reply:
@aaronc just use `appCodec` now.